File: 程式語言 V2答案檔/Go/210.go
package main
import (
	"fmt"
	"math"
)

func main(){
	var a, b int
	fmt.Scanln(&a)
	fmt.Scanln(&b)
	gcd := 1
	for i := 1; i <= int(math.Min(float64(a), float64(b))); i++{
		if a % i == 0 && b % i == 0 {
			gcd = i
		}
	}
	lcm := (a * b) / gcd
	fmt.Println(gcd)
	fmt.Println(lcm)
}

File: 程式語言 V2答案檔/C#/210.cs
using System;

public class TQC210
{
    public static void Main(string[] args)
    {
        int a, b, gcd, lcm;
        a = int.Parse(Console.ReadLine());
        b = int.Parse(Console.ReadLine());
        gcd = ComputeGCD(a, b);
        lcm = ComputeLCM(a, b, gcd);

        Console.WriteLine(gcd);
        Console.WriteLine(lcm);
    }

    public static int ComputeGCD(int a, int b) {
        if (b == 0)
            return a;
        return ComputeGCD(b, a % b);
    }

    public static int ComputeLCM(int a, int b, int gcd) {
        return Math.Abs(a * b) / gcd;
    }
}